systemd/systemd.go

```diff
@@ -358,6 +358,25 @@ type Systemd interface {
 	// threads if enabled, etc) part of the unit, which can be a service or a
 	// slice.
 	CurrentTasksCount(unit string) (uint64, error)
+	// Run a command
+	Run(command []string, opts *RunOptions) ([]byte, error)
+}
+
+// KeyringMode describes how the kernel keyring is setup, see systemd.exec(5)
+type KeyringMode string
+
+const (
+	KeyringModeInherit KeyringMode = "inherit"
+	KeyringModePrivate KeyringMode = "private"
+	KeyringModeShared  KeyringMode = "shared"
```

Comment:
nitpick: the constants' names could be simplified so they don't "stutter" (the type, variable and field names are already going to be named KeyringMode). It seems slightly cleaner but feel free to ignore if you disagree.
```suggestion
	Inherit KeyringMode = "inherit"
	Private KeyringMode = "private"
	Shared KeyringMode = "shared"
```